Add multiple CD pipeline selector docs (#1815)
wso2/docs-choreo-dev: en/developer-docs/docs/devops-and-ci-cd/manage-continuous-deployment-pipelines.md

    !!! info "Note"
        The **default** continuous deployment pipeline is configured separately at both the organization and project levels. When a project is created, it inherits the organization's **default** pipeline. The project's **default** pipeline then defines the default promotion order for its components on the Deploy page.

## Use multiple continuous deployment pipelines within a component

### Prerequisites

> - To use multiple continuous deployment pipelines in the deploy page of a component, you must have more than 1 continuous deployment pipelines assigned to the project of that component.

Reviewer: ```suggestion
- To use multiple continuous deployment pipelines on a component's deploy page, the component's project must have more than one continuous deployment pipeline assigned.
```
WDYT?

> To use multiple continuous deployment pipelines in a component follow the steps given below,

Reviewer: ```suggestion
To use multiple continuous deployment pipelines in a component, follow the steps given below:
```

1. Sign in to [Choreo](https://console.choreo.dev/).

> 2. In the Choreo Console, go to the top navigation menu and click **Organization**. Then select your organization.
> 3. Click the project that has the component you want to use multiple continuous deployment pipelines.
> 4. Click the component you want to use multiple continuous deployment pipelines.

Reviewer: ```suggestion
2. In the Choreo Console top navigation menu, select the Organization, then the Project and finally the Component where you want to use multiple continuous deployment pipelines.
```
Can we say it like this?

5. In the left navigation menu, click **Deploy**.

> 6. At the top of the page, you can see a dropdown next to the Deployment Track/Version dropdown. This dropdown allows you to select the continuous deployment pipeline you want to use to deploy the component.

Reviewer: ```suggestion
6. At the top of the page, click on the dropdown next to the Deployment Track/Version dropdown and choose the continuous deployment pipeline you want to use to deploy the component.
```

    !!! info "Note"

>         The user will be able to see all the continuous deployment pipelines assigned to the project and select one of them to deploy the component. Existing permissions will be enforced to allow deployment and promotion of the component for specific environments.

Reviewer: ```suggestion
        The user will be able to view all continuous deployment pipelines assigned to the project and select one to deploy the component. Existing permissions will continue to control deployment and promotion access for specific environments.
```
